api/profiles/dev/application.properties

```diff
@@ -159,4 +159,8 @@ log.security.elastic.index.prefix=${CP_SECURITY_LOGS_ELASTIC_PREFIX:security_log
 migration.alias.file=${CP_API_MIGRATION_ALIAS_FILE:}
 
 #Cache
-cache.type=MEMORY
\ No newline at end of file
+cache.type=MEMORY
+
+#edge
+edge.internal.host=${CP_EDGE_INTERNAL_HOST:cp-edge.default.svc.cluster.local}
```

Comment:
We also need to add these new properties to `deploy/docker/cp-api-srv/config/application.properties`